### PR TITLE
Created basic functionality for User page

### PR DESCRIPTION
**Pull request by aOrro**

### CODE REVIEW

File: src/components/UserHeader/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export const UserHeader = props => {
+  const userInfo = props.userInfo;
+
```
Comment (yoloOnTheBattlefield) [inline src/components/UserHeader/index.js:3]: destructure

File: src/components/UserHeader/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      </div>
+      <div>
+        <h2>{userInfo && userInfo.name}</h2>
+        <span>{userInfo && userInfo.username}</span>
+        <span>{userInfo && userInfo.bio}</span>
+        <span>{userInfo && userInfo.location}</span>
+        <span>{userInfo && userInfo.total_photos} Photos</span>
+        <span>{userInfo && userInfo.followers_count} Followers</span>
+        <span>{userInfo && userInfo.following_count} Following</span>
+      </div>
```
Comment (yoloOnTheBattlefield) [inline src/components/UserHeader/index.js:20]: this is quite repetitive, can it be done better?

File: src/components/UserStats/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+export const UserStats = props => {
+  return (
+    <div>
+      <h2 onClick={props.click}>Stats</h2>
```
Comment (yoloOnTheBattlefield) [inline src/components/UserStats/index.js:4]: handleClick please

File: src/pages/User/index.js [hunk 4]
```diff
@@ -1,3 +1,105 @@
...
+      console.log(error);
+    }
+  };
+
+  getUserStats = async () => {
+    try {
+      const { data } = await axios(
+        `https://api.unsplash.com/users/${this.state.inputValue}/statistics?client_id=${process.env.REACT_APP_API_KEY}`
+      );
+      console.log(data);
```
Comment (yoloOnTheBattlefield) [inline src/pages/User/index.js:61]: no console

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/UserHeader/index.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        />
+      </div>
+      <div>
+        <h2>{userInfo && userInfo.name}</h2>
+        <span>{userInfo && userInfo.username}</span>
+        <span>{userInfo && userInfo.bio}</span>
+        <span>{userInfo && userInfo.location}</span>
+        <span>{userInfo && userInfo.total_photos} Photos</span>
+        <span>{userInfo && userInfo.followers_count} Followers</span>
+        <span>{userInfo && userInfo.following_count} Following</span>
```
Comment (redliz5808) [inline src/components/UserHeader/index.js:19]: Could you do one check for userInfo here to make it more concise?

File: src/pages/Search/index.js [hunk 6]
```diff
@@ -16,13 +16,11 @@ class Search extends React.Component {
       this.setState({
         isLoading: true,
       });
-      const apiKey = 'udklURXoWXmXIiykDuO2luB1M4X-HNkuOePzmJmjMhc';
       const { inputValue, pageToLoad, photosData } = this.state;
       const { data } = await axios(
-        `https://api.unsplash.com/search/photos?page=${pageToLoad}&query=${inputValue}&client_id=${apiKey}`
+        `https://api.unsplash.com/search/photos?page=${pageToLoad}&query=${inputValue}&client_id=${process.env.REACT_APP_API_KEY}`
```
Comment (vladciuca) [inline src/pages/Search/index.js:21]: ${process.env.REACT_APP_API_KEY} maybe you could move this in its own variable then pass it in the url

File: src/pages/User/index.js [hunk 7]
```diff
@@ -1,3 +1,105 @@
...
+    userInfo: null,
+    userPhotos: [],
+    userCollections: [],
+    userStats: null,
+  };
+
+  getUserInfo = async () => {
+    try {
+      const { data } = await axios(
+        `https://api.unsplash.com/users/${this.state.inputValue}?client_id=${process.env.REACT_APP_API_KEY}`
```
Comment (vladciuca) [inline src/pages/User/index.js:20]: same here for ${process.env.REACT_APP_API_KEY}

File: src/pages/User/index.js [hunk 8]
```diff
@@ -1,3 +1,105 @@
...
+      });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  getUserCollections = async () => {
+    try {
+      const { data } = await axios(
+        `https://api.unsplash.com/users/${this.state.inputValue}/collections?page=1&per_page=10&client_id=${process.env.REACT_APP_API_KEY}`
```
Comment (vladciuca) [inline src/pages/User/index.js:46]: here too

File: src/pages/User/index.js [hunk 9]
```diff
@@ -1,3 +1,105 @@
...
+      });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  getUserStats = async () => {
+    try {
+      const { data } = await axios(
+        `https://api.unsplash.com/users/${this.state.inputValue}/statistics?client_id=${process.env.REACT_APP_API_KEY}`
```
Comment (vladciuca) [inline src/pages/User/index.js:59]: here too

**Review by vladciuca — APPROVED** [review]

**Pull request merged**